wiki/BBCode/de.md

```diff
@@ -0,0 +1,304 @@
+
+*Nicht zu verwechseln mit [Spoilerbox](#spoilerbox).*
+
+```
+[box=NAME]
+Text
+[/box]
+```
+
+Das Tag `[box]` wird verwendet, um Texte und Bilder in einem anklickbaren Hyperlink zu verstecken. Beim Klicken wird der Inhalt in der Box ähnlich wie in einem Dropdown-Menü aufgedeckt.
+
+Der benutzerdefinierte Hyperlink wird durch das Argument `NAME` bestimmt. Die Spezifizierung dieses Arguments wird eine Überschrift in der Box anhand des Arguments erstellen und wird die Größe der Box entsprechend anpassen. Wenn die Spezifizierung nicht erfolgt, dann wird das Tag `[box]` standardmäßig eine Box ohne Überschrift erstellen (auf die nicht geklickt werden kann). Das Argument benutzt keine Anführungszeichen (`"`) und stellt Leerzeichen dar.
+
+Das Tag wird meistens benutzt, um große Fluten an Texten und Bildern zu verstecken, die den Forum-Beitrag immens verlängern. Am meisten sichtbar in den FAQ oder in [Skin](/wiki/Skinning)-Veröffentlichungsbeiträgen.
+
+*Tipp: die Schaltfläche der BBCode-Box in der Symbolleiste wird "spoiler box" genannt, aber erstellt keinen `[spoilerbox]` Tag.*
+
+Schaltfläche in der Symbolleiste: ![Box-Button](img/spoilerbox.png "Box")
+
+### Spoilerbox
+
+```
+[spoilerbox]Text[/spoilerbox]
+```
+
+Eine *Spoilerbox* ist eine spezielle Art von BBCode-Box, die kein spezifizierbares Argument `NAME` hat. Der Name einer Spoilerbox wird immer als `SPOILER` angezeigt. Spoilerboxen haben ihr eigenes Tag (`[spoilerbox]`), aber sind funktional identisch zu BBCode-[Boxen](#box).
+
+### Zitat
+
+```
+[quote="NAME"]
+Text
+[/quote]
+```
+
+Das Tag `[quote]` wird benutzt, um lange Zitate stylistisch durch Einrückung, Färbung, Fettschrift und einer pinken vertikalen Linie zu formatieren. Der tatsächliche Inhalt des Zitats wird zwischen dem öffnenden und dem schließenden Tag platziert, während das Argument `NAME` den Autor des Zitats spezifiziert (das ist optional). Der Text innerhalb des Zitats stellt Leerzeichen und Zeilenumbrüche dar.
+
+*Hinweis: Das Argument `NAME` muss zwischen Anführungszeichen (`"`) gesetzt werden.*
+
+Lange Zitate werden typischerweise in formalen Schreiben bei Texten verwendet, die länger als zwei Zeilen sind. Innerhalb der osu!-Foren werden sie allerdings meistens genutzt, um auf den Kommentar eines anderen Nutzers zu antworten. Das kann über den Button `Beitrag in der Antwort zitieren` in der rechten oberen Ecke des entsprechenden Kommentars gemacht werden (unten gezeigt). Dieser Button wird **nur erscheinen, wenn der Cursor in der Nähe ist**.
+
+![Zitat-Button](img/quotereply.png)
+
+### Codeblock
+
+```
+[code]
+Text
+[/code]
+```
+
+Das Tag `[code]` wird benutzt, um *vorformatierte Codeblöcke* (auch *vorformatierter Text*) zu erstellen. Auf der osu!-Webseite formatiert dieses Tag Text in der Schriftart Mono Space innerhalb einer halbtransparenten grauen Box. Die Formatierung von Text innerhalb eines Codeblocks behandelt der Editor als reinen Text, sodass dieser nicht als Quellcode oder ähnliches interpretiert wird.
```

Comment:
```suggestion
Das Tag `[code]` wird benutzt, um *vorformatierte Codeblöcke* (auch *vorformatierter Text*) zu erstellen. Auf der osu!-Webseite formatiert dieses Tag Text in einer Monospace-Schriftart innerhalb einer halbtransparenten grauen Box. Die Formatierung von Text innerhalb eines Codeblocks behandelt der Editor als reinen Text, sodass dieser nicht als Quellcode oder ähnliches interpretiert wird.
```

Monospace isn't a single font, it's a type of font where every character takes the same amount of space.